test/e2e/azure_suite_test.go

```diff
@@ -116,7 +116,7 @@ var _ = BeforeSuite(func() {
 	waitDeployment(c, "cert-manager", "cert-manager-webhook")
 
 	// Deploy the CAPI and CABPK components from Cluster API repository,
-	capi := &generators.ClusterAPI{Version: "v0.3.0-rc.3"}
+	capi := &generators.ClusterAPI{Version: "v0.3.0"}
 	cabpk := &generators.Bootstrap{Version: "v0.1.5"}
```

Comment:
can we also change this to `v.0.3.0` since its actually getting the bootstap from cluster-api

Reply:
If a separate call is needed, yes it should be updated

Reply:
updated